### PR TITLE
doc: doc fixes related to NCSDK-9787

### PR DESCRIPTION
**Pull request by umapraseeda**
Remove occurences of 'please' from NCS docs
ref: NCSDK-9787

Signed-off-by: Uma Praseeda <uma.praseeda@nordicsemi.no>

### CODE REVIEW

File: crypto/nrf_cc310_mbedcrypto/CHANGELOG.rst [hunk 1]
```diff
@@ -916,7 +916,7 @@ New experimental version of nrf_cc310_mbedcrypto with changes to platform initia
 
 ..warning::
    This version may lead to undefined behavior on some nRF9160 devices.
-   Please use a newer version.
+   Hence, use a newer version.
```
Comment (peknis) [inline crypto/nrf_cc310_mbedcrypto/CHANGELOG.rst:919]: Change the warning to a note. You remember, no warnings unless someone's health or life is at risk.

File: softdevice_controller/README.rst [hunk 2]
```diff
@@ -43,8 +43,8 @@ Proprietary feature support:
 | QoS Conn Event Reports   | X               | X            | X         |
 +--------------------------+-----------------+--------------+-----------+
 
-.. note::
-   Please note the following when using the library for the nRF5340:
+
+Note the following when using the library for the nRF5340:
```
Comment (peknis) [inline softdevice_controller/README.rst:48]: Why not just a note as follows:

`.. note::`
    Low Latency Packet mode is not supported on the nRF53 Series.

File: crypto/nrf_cc310_platform/CHANGELOG.rst [hunk 3]
```diff
@@ -578,17 +578,19 @@ Added a new build of nrf_cc3xx_mbedcrypto libraries for nRF9160, nRF52840, and n
...
-Added new version of libraries nrf_cc310_platform/nrf_cc312_platform built with mbed TLS version 2.23.0
+Added new version of libraries nrf_cc310_platform/nrf_cc312_platform built with mbed TLS version 2.23.0.
 
-Added APIs for storing keys in the the KMU peripheral (nRF9160, nRF5340). Please see include/nrf_cc3xx_platform_kmu.h
+Added APIs for storing keys in the the KMU peripheral (nRF9160, nRF5340).
+See :file:`include/nrf_cc3xx_platform_kmu.h`.
 
-Added APIs for generating CSPRNG using CTR_DRBG. Please see include/nrf_cc3xx_platform_ctr_drbg.h
+Added APIs for generating CSPRNG using CTR_DRBG.
+See :FILE:`include/nrf_cc3xx_platform_ctr_drbg.h`.
```
Comment (peknis) [inline crypto/nrf_cc310_platform/CHANGELOG.rst:591]: FILE -> file

File: crypto/nrf_cc310_platform/CHANGELOG.rst [hunk 4]
```diff
@@ -578,17 +578,19 @@ Added a new build of nrf_cc3xx_mbedcrypto libraries for nRF9160, nRF52840, and n
...
 **************************
 
-Added experimental support for devices with Arm CryptoCell CC312 (nRF5340)
+Added experimental support for devices with Arm CryptoCell CC312 (nRF5340).
 
-Changed name of configurations from CC310 to CC3XX. This is reflected in the header file and APIs as well, where nrf_cc310_xxxx is renamed to nrf_cc3xx_xxxx
+Changed name of configurations from CC310 to CC3XX. This is reflected in the header file and APIs as well, where nrf_cc310_xxxx is renamed to nrf_cc3xx_xxxx.
 
-Added new version of libraries nrf_cc310_platform/nrf_cc312_platform built with mbed TLS version 2.23.0
+Added new version of libraries nrf_cc310_platform/nrf_cc312_platform built with mbed TLS version 2.23.0.
```
Comment (peknis) [inline crypto/nrf_cc310_platform/CHANGELOG.rst:585]:
```suggestion
Added new version of libraries nrf_cc310_platform/nrf_cc312_platform built with Mbed TLS version 2.23.0.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: crypto/nrf_cc310_platform/CHANGELOG.rst (hunk 4 above)
Comment (umapraseeda) [reply crypto/nrf_cc310_platform/CHANGELOG.rst:585]: will be fixed in PR:509

**Review by peknis — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Review by joerchan — APPROVED** [review]

**Review by rugeGerritsen — APPROVED** [review]
SDC changes are ok.

**Pull request merged**